app/views/checkout/_summary.html.haml

```diff
@@ -1,35 +1,29 @@
 %orderdetails
-  = form_for current_order, url: "#", html: {"ng-submit" => "purchase($event, checkout)"} do |f|
```

Comment:
Good question. I guess technically it doesn't need to be a form anymore. Is anything calling the submit action on the form?

So if it's not a form, you need to update the stylesheets and the test cases.